godot-codegen/src/generator/enums.rs

```diff
@@ -105,9 +107,28 @@ pub fn make_enum_definition_with(
 
         // Trait implementations.
         let engine_trait_impl = make_enum_engine_trait_impl(enum_, enum_bitmask.as_ref());
+        let property_hint = if enum_.is_bitfield {
+            quote! { crate::global::PropertyHint::FLAGS }
+        } else {
+            quote! { crate::global::PropertyHint::ENUM }
+        };
         let index_enum_impl = make_enum_index_impl(enum_);
         let bitwise_impls = make_enum_bitwise_operators(enum_, enum_bitmask.as_ref());
 
+        let var_trait_set_property = if enum_.is_exhaustive {
+            quote! {
+                fn set_property(&mut self, value: Self::Via){
```

Comment:
```suggestion
                fn set_property(&mut self, value: Self::Via) {
```

Please fix this same formatting error everywhere 🙂